project/Dependencies.scala

```diff
@@ -90,6 +90,23 @@ object Dependencies {
     ExclusionRule("javax.servlet", "javax.servlet-api")
   )
 
+  val defaultWithMesos = sys.props.getOrElse("with.mesos", "false").toBoolean
```

Comment:
let's put it true by default? Is it a lot of extra deps?

If it's not by default here, I'll have to change the generator ... and I'll put it by default at true rather than changing the UI ... :-)
Also, Yarn is there by default isn't it?